xml/deployment_firstboot.xml

```diff
@@ -0,0 +1,870 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//Novell//DTD NovDoc XML V1.0//EN" "novdocx.dtd"
+[
+  <!ENTITY % NOVDOC.DEACTIVATE.IDREF "INCLUDE">
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
```

Comment:
This should be just a minimal DOCTYPE without the reference to Novdoc. Something like this:

```suggestion
<!DOCTYPE chapter 
[
  <!ENTITY % entities SYSTEM "entity-decl.ent">
  %entities;
]>
```